include/cantera/kinetics/Kinetics.h

```diff
@@ -555,6 +558,15 @@ class Kinetics
      * @param i   reaction index
      */
     virtual double reactantStoichCoeff(size_t k, size_t i) const;
+
+    /**
+     * Stoichiometric coefficient matrix for reactants.
+     */
+    Eigen::SparseMatrix<double> reactantStoichCoeffs() const
+    {
+        return m_reactantStoich.stoichCoeffs();
+    }
```

Comment:
For these one liners, I think the following style is fine (and consistent with the rest of this file):
```suggestion
    Eigen::SparseMatrix<double> reactantStoichCoeffs() const {
        return m_reactantStoich.stoichCoeffs();
    }
```

Reply:
:+1: ... I was under the impression that for methods the initial `{` should preferably be on its own line and hence started to apply this whenever touching code. I (personally) prefer what you proposed anyways and will adopt. 

So ... the initial `{` will only be put on its own line if the argument list doesn't fit on the first line?